config/features.rb

```diff
@@ -3,4 +3,11 @@
   strategy :default
 
   feature :test_feature, description: "This is a test feature"
+
+  group :placements do
+    feature :user_onboarding_emails,
+            description: "Dispatch emails to users when they have been" \
+            " onboarded onto the placements service",
+            default: false
```

Comment:
This description doesn't provide the context that it also effects "removed" email notifications.

```suggestion
            description: "Dispatch emails to users when they have been" \
            " onboarded to or removed from the placements service",
            default: false
```

Reply:
resolved in [f45edab](https://github.com/DFE-Digital/itt-mentor-services/pull/523/commits/f45edabc57529e8167401990eb146c5a604d3033)